src/main/java/at/hannibal2/skyhanni/api/GetFromSackAPI.kt

```diff
@@ -165,31 +165,42 @@ object GetFromSackAPI {
     )
 
     private fun commandValidator(args: List<String>): Pair<CommandResult, PrimitiveItemStack?> {
-        if (args.size <= 1) {
-            return CommandResult.WRONG_ARGUMENT to null
+        if (args.isEmpty()) return CommandResult.WRONG_ARGUMENT to null
+
+        val arguments = args.toMutableList()
+        val argsNull = args.last().toIntOrNull() == null
```

Comment:
this could work for the calculate issue
```suggestion
        val argsNull = args.last().last().toString().toIntOrNull() == null
```

Reply:
Actually idk if String supports .last() . Nevertheless it isn't that hard to implement